corehq/apps/locations/migrations/0006_auto_20161114_1858.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.11 on 2016-11-14 18:58
+from __future__ import unicode_literals
+
+from django.db import migrations
+import django.db.models.manager
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('locations', '0005_locationtype_include_without_expanding'),
+    ]
+
+    operations = [
+        migrations.AlterModelManagers(
+            name='sqllocation',
+            managers=[
+                ('_tree_manager', django.db.models.manager.Manager()),
```

Comment:
Thanks for flagging this.

I think this is actually a bug that was released in 1.9 and it has been fixed in 1.10 https://code.djangoproject.com/ticket/26643 I'm not able to test it though as hq doesn't run on 1.10 yet

I don't think it will cause issues either way. It just changes the manager that is used during migrations